Resources/Prototypes/Entities/Structures/Holographic/projections.yml

```
@@ -117,18 +120,22 @@
             - TableMask
           layer:
             - TableLayer
-    - type: TimedDespawn
+    #- type: TimedDespawn # DeltaV - Replaced by TimedDespawnDetailed
+    #  lifetime: 180
+    - type: TimedDespawnDetailed # DeltaV
       lifetime: 180
+    - type: Holobarrier # DeltaV - Whitelist for holoprojector pickup
     - type: PointLight
       enabled: true
       radius: 3
       color: red
     - type: Climbable
     - type: Clickable
 
+
 - type: entity
```

Remove new blank line
```suggestion

- type: entity
```